keras_cv/layers/preprocessing/rand_augment.py

```diff
@@ -58,7 +58,10 @@ class RandAugment(RandomAugmentationPipeline):
             Defaults to `0.15`.
         rate:  the rate at which to apply each augmentation.  This parameter is applied
             on a per-distortion layer, per image.  Should be in the range `[0, 1]`.
-            Defaults to `1`.
+            Defaults to `10/11`, as the original `RandAugment` paper includes an
```

Comment:
Not 100% sure we should do it. This adds overhead, and the actual benefits are unclear. We could simply leave it to 1.0 and mention that 10 / 11 reproduces the implementation proposed by the paper.

Reply:
Done - yeah, this is kind of weird and you are right that it is unclear if it actually even helps.

Reply:
The ablation study was done only for  Wide-ResNet-28-2 models weretrained on CIFAR-10 using RandAugment (N = 3, M = 4)

![immagine](https://user-images.githubusercontent.com/1710528/163041048-aa4e8121-0f9b-4f68-bd32-485733ecedf0.png)

Do we need to remove negative avg augmentations from the list? Do we know if this was confirmed for the other datasets/models?

Reply:
As always.. too late :)

Reply:
> The ablation study was done only for `Wide-ResNet-28-2` models trained on `CIFAR-10` using `RandAugment (N = 3, M = 4)`
> 
> ![immagine](https://user-images.githubusercontent.com/1710528/163041048-aa4e8121-0f9b-4f68-bd32-485733ecedf0.png)
> 
> Do we need to remove negative avg augmentations from the list? Do we know if this was confirmed for the other datasets/models?

Hey @bhack - it is not confirmed for other datasets.  I discussed this with the original paper author.  He highly recommended the set of augmentations we now have.

Reply:
Ok thanks

Reply:
> Ok thanks

I was very confused by the presence of negative values in the table in the paper too.  It is very confusing with respect to the point they're trying to show (that this is a good general purpose augmentation technique)

Reply:
> I was very confused by the presence of negative values in the table in the paper too. It is very confusing with respect to the point they're trying to show (that this is a good general purpose augmentation technique)

Yes I suppose that also a more trivial augmentation policy like TA could perform better in some cases:

![immagine](https://user-images.githubusercontent.com/1710528/163055781-4c72f7fa-638f-41b2-9903-5d9e595e6100.png)